app/scripts/views/PlacesNew.js

```diff
@@ -63,7 +65,7 @@ class PlacesNew extends Component {
       location: {
         mode: DEFAULT_MODE,
         street: '',
-        city: config.defaultCity,
+        city: '',
```

Comment:
What is the reasoning behind removing the default city here?

Reply:
Ah, I did this because not all events will be in London, so I don't wanna suggest London for everything by default. But actually, maybe we could just set defaultCity to optional in the config form / model?

Reply:
Good idea! I like that better! Its still nice for convenience to have a default set (optionally) as some festivals might just be in one city

Reply:
Let's merge this PR and I'll create a new one for making defaultCity optional.